package.json

```diff
@@ -31,7 +31,7 @@
     "gulp-sourcemaps": "1.5.2",
     "gulp-uglify": "1.1.0",
     "gulp-util": "3.0.6",
-    "jest-cli": "0.4.18",
+    "jest-cli": "*",
```

Comment:
don't commit this update, the rest of the pbw team is using a previous version of node that is compatible with 0.4.18